typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts

```diff
@@ -36,6 +34,21 @@ const chainsToDeploy = [
 ];
 export const MAX_PROTOCOL_FEE = parseEther('100').toString(); // Changing this will redeploy the PROTOCOL_FEE hook
 
+const tokenPrices: ChainMap<string> = {
```

Comment:
worth having a quick test to ensure every chain in chainsToDeploy has a tokenPrice?